controllers/clients.js

```diff
@@ -1,135 +1,72 @@
-const fs = require('fs');
-const clients = require('../data/clients.json');
+const Clients = require('../models/Clients');
 
-const errorResHelper = (errorDescription, res, errCode = 400) => {
-  res.status(errCode).json({ error: errorDescription });
+const listClients = (req, res) => {
+  Clients.find(req.query)
+    .then((clients) => res.status(200).json(clients))
+    .catch((error) => res.status(400).json({ message: error }));
 };
 
 const createClient = (req, res) => {
-  const now2ISO = new Date().toISOString();
-  const missing = [];
-  const newClient = {
-    id: new Date().getTime().toString(),
-    name: req.query.name ?? missing.push("'name'"),
-    phone: req.query.phone ?? missing.push("'phone'"),
-    location: {
-      country: req.query.country ?? missing.push("'country'"),
-      state: req.query.state ?? missing.push("'state'"),
-      city: req.query.city ?? missing.push("'city'"),
-      address: req.query.address ?? missing.push("'address'"),
-    },
-    description: req.query.description ?? missing.push("'description'"),
-    logo: req.query.logo ?? missing.push("'logo'"),
+  const clientCreated = new Clients({
+    name: req.body.name,
+    phone: req.body.phone,
+    location: req.body.location,
+    logo: req.body.logo,
+    description: req.body.description,
     created: {
-      idAdmin: req.query.idAdmin ?? missing.push("'idAdmin'"),
-      timestamp: now2ISO,
+      admin: req.body.created.admin,
+    },
+    modified: {
+      admin: req.body.modified.admin,
```

Comment:
it is not necessary since you set timestamp in the Schema

Reply:
Ok, removed it